common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -416,6 +416,18 @@ private static byte ipv4WordToByte(String ip, int from, int toExclusive) {
         };
     }
 
+    /**
+     * Convert {@link Inet4Address} into {@code int}
+     */
+    public static int ipv4AddressToInt(Inet4Address ipAddress) {
```

Comment:
when you say we do this quite frequently what does this mean ?

Reply:
I actually mean, we are converting IPv4Address into Integer quite often like in PR #10492 or #10498. In PCapWriter, we convert IPv4Address to Integer to write into Pcap file and in Binary search one, we need Integer Ip address not `InetAddress`. Even in unit tests. So it might be a good idea to push that code into `NetUtils` instead of copy-pasting every time.

Reply:
got it...